### PR TITLE
landing-page-mobile

### PR DESCRIPTION
**Pull request by DmitryVolchokFE**
[DEMO LINK](https://dmitryvolchokfe.github.io/layout_landing-page/)

### CODE REVIEW

**Review by OleziO — CHANGES_REQUESTED** [review]
image
Fix demo

**Review by OleziO — CHANGES_REQUESTED** [review]
.

**Comment by DmitryVolchokFE** [comment]
soorry, new link (https://dmitryvolchokfe.github.io/layout_landing-page)

**Review by demetrxx — CHANGES_REQUESTED** [review]
Size and position of many images don't match design at all. 
Hero img isn't centered. 
Compare bikes list items items should be in one row.
Contact us section layout doesn't match the design
Footer has wrong height.

Please review your layout and make sure it, at least, looks like in the provided design. 
Waiting for your fixes.

**Comment by DmitryVolchokFE** [comment]
Good afternoon, I submit this page for verification, in the task: https://mate.academy/learn/css-advanced/bem?section=video&videoId=4559&testTaskSlug=markup_landing_page
At the end of the lesson, it was indicated to create a mobile page, and to adapt the menu and header to the tablet. I did it. There is no adaptation for PC. (https://dmitryvolchokfe.github.io/layout_landing-page)

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,329 @@
...
+          >
+        </a>
+
+        <div class="top-bar__icons">
+          <div class="icon">
+            <a href="tel:+1 234 5555-55-55" class="icon__position icon--phone"></a>
+          </div>
+
+          <div class="icon">
+            <a href="#menu" class="icon__position icon--menu"></a>
```
Comment (anastasiiavorobiova) [inline src/index.html:35]: Consider using the aria-label attribute to explain the purpose of this empty link. It has a significant impact on accessibility of this page.

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,329 @@
...
+        <a href="#home">
+          <img
+            src="images/logo-black.svg"
+            alt="MyBiKE logo"
+            class="top-bar__logo--black"
+          >
+        </a>
+
+        <div class="icon--close_menu">
+          <a href="#" class="icon__position icon--close"></a>
```
Comment (anastasiiavorobiova) [inline src/index.html:55]: Consider using the aria-label attribute to explain the purpose of this empty link. It has a significant impact on the accessibility of this page.

File: src/styles/blocks/contacts.scss [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    font-weight: 500;
+    line-height: 22.4px;
+  }
+
+  &__info:hover {
+    color: #ddd;
+  }
+
+  &__info:active {
+    color: #bbb;
```
Comment (anastasiiavorobiova) [inline src/styles/blocks/contacts.scss:50]: Consider using variables for this

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work!
See comments to improve it a bit.

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
1. image

2. add favicon 
image

3. fix button width 
image

4. all images should be scaled on the hover 
image

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
1. images should not overlap each other
image


2. Page should not reloads or lead to 405 page on submit, also clear form fields on button click
image

If you have any questions - feel free to ask them in fe_chat

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Add transition for this button when on hover and remove default borders for this element
image
image
Fix this problem when on hover
image

**Comment by DmitryVolchokFE** [comment]
Recorded all comments

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Not everything has been fixed since the last review, check the comments and fix them

**Comment by DmitryVolchokFE** [comment]
Then I did not fully understand what was meant, corrected.

**Review by Volodymyr-Melnyk-Fron10der — APPROVED** [review]

**Pull request closed without merging**